content/community/events.page.tsx

```diff
@@ -170,20 +167,19 @@ const events = [
     link: "https://www.meetup.com/xrp-ledger-miami-community/events/284463736/",
     location: "The LAB Miami, FL",
     date: "April 6th 2022",
-    image: "event-meetup-miami@2x.jpg",
+    image: require("../static/img/events/event-meetup-miami@2x.jpg"),
     end_date: "April 6, 2022",
   },
-
   {
-    name: "Hackathon:New Year, New NFT",
+    name: "Hackathon:<br />New Year, New NFT",
```

Comment:
Redocly doesn't support HTML tags in messages like this.

![2024-01-08_150910_370618724](https://github.com/XRPLF/xrpl-dev-portal/assets/7515597/e8d07f7e-59fa-47b3-901e-28066bb4934f)


```suggestion
    name: "Hackathon: New Year, New NFT",
```